Dockerfile

```
@@ -1,22 +1,56 @@
-# Java 21 런타임 (클래스 버전 65)
-FROM eclipse-temurin:21-jre-alpine
+plugins {
```

Hadolint (2.14.0)

[error] unexpected 'p'
expecting '#', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, end of input, or whitespaces

(DL1000)